SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -389,10 +389,12 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(Config)
 	bool bEnableCrossLayerActorSpawning;
 
-	// clang-format off
+	/**
+	 * Whether or not to suppress a warning if an RPC of Type is being called with unresolved references. Default is false.
+	 * QueuedIncomingWaitRPC time is still respected.
+	 */
 	UPROPERTY(EditAnywhere, Config, Category = "Logging", AdvancedDisplay,
-		meta = (DisplayName = "Whether or not to suppress a warning if an RPC of Type is being called with unresolved references. Default is false.  QueuedIncomingWaitRPC time is still respected."))
-	// clang-format on
+			  meta = (DisplayName = "RPCTypes to allow unresolved parameters for"))
```

Comment:
"RPCTypes to allow unresolved parameters for"
? Not sure what this means?

Reply:
Also not sure what would be the best name for this.
What I wanted to convey is "Specify the RPCTypes for which unresolved parameters are not an issue for"

Reply:
That sounds better

Reply:
Hmm, actually kind of forgot to address this, and wanted to merge.
Don't want to name it exactly "Specify the RPCTypes for which unresolved parameters are not an issue for", since having a verb (Specify) as the setting name seems weird.

Reply:
RPCTypes *that* allow unresolved parameters ?